### ISSUE TITLE
Ideas for 3.0.4 release

### ISSUE THREAD

**Issue by matthijskooijman** [projecthamster/hamster#748]
So, the last week or so was a nice spurt of development. I very much enjoyed the combined effort of quite a few people leading up to the 3.0.3 release, so thanks all for your contributions!

So, @DirkHoffmann and I discussed we should try to keep some of this momentum for some followup improvements, and maybe try to do another release soon (maybe around Christmas)? So let's exchange some ideas on what should be in the next release?

One thing that I would like to improve is documentation. Both for the user (Like clarifying the install options in the README) as well as for the developer (development and release policies, etc.). There are already issues for most of these:
 - #660
 - #585
 - #608
 - #611

While preparing for the last release, I've noticed some things that could make the release and developer experience easier and I would like to see improved. I've created separate issues for those:
 - #743
 - #744
 - #747
 - #746
 - #745

On the functionality side, I still have #627 / #630 on my radar (some of which is fixed, but not all). Also, #702 is annoying and would be good to finish up.

There has also been some talk about revamping the build system (#740) and translations (#741), but those are probably way too big for a short timeline (so they should probably not delay a next release, but that shouldn't hinder discussion or work on them either).

So, that's my thoughts, which are probably overfocused on the things I have been working on (and certainly already too much too all tackle before christmas). Let me hear your thoughts about what would be (not) worth working on next?

Finally, as an off-topic note - We've been using the #projecthamster IRC channel (on libera.chat) again this weekend. If you still have a working IRC client, feel free to drop by :-)

**Comment by DirkHoffmann** [comment]
Nice to see that my modest effort helped to give the project some speed. Yes, there were quite some people becoming active (again) last week and mostly contributed constructively and in the background.

Your topics/priorities are fine for me, @matthijskooijman. And I agree that it may be a bit ambitious to work on all of them. Maybe you can use the assignment of tickets to indicate on what you are working. That makes it easier for others, to see where help can be needed.

You mentioned only one pull request out of 15 (as of today) pending. How do you see those?

**Comment by matthijskooijman** [comment]
> You mentioned only one pull request out of 15 (as of today) pending. How do you see those?

Good point. I think we should try to welcome contributions by investing in people's pullrequests. I think over the past time we've merged quite a few of them already, I guess those were the easy ones that required limited work or discussion and the ones left might be somewhat more complex or requiring some decision.

Above, I've already mentioned #630 and #702. #690 is probably going to be included in #743. Then #720 seems to be worked on and maybe reaching a point where it can be merged. #616 and #617 are related and have seen some discussion, but need new review and probably a decision on the approach used. #626 is too big and unpolished currently, and has been waiting for the PR submitter to reply for 3 years. I have not really looked at the older ones yhet.

**Comment by aquaherd** [comment]
Although I see the need for new development, I also emphasize the need for infrastructure improvement.

I have successfully packaged hamster for `alpine` but given up waiting for `void` to pick my PR.
We could do with some improved visibility for packagers, aim for a pip release, pick up appimage builds and maybe even feature a repology badge at our landing page:

[![Packaging status](https://repology.org/badge/vertical-allrepos/hamster-time-tracker.svg)](https://repology.org/project/hamster-time-tracker/versions)

As can be seen here, getting a foot back into `fedora` should be a major goal.

**Comment by matthijskooijman** [comment]
@aquaherd, thanks for pitching in. It seems we are in agreement to focus on development tools and porting rather than new features, which I think is also the gist or my list of suggested issues.

In the meanwhile, I've merged a fix for the issues described in #747. I've been working on a `--replace-all` option (see #751) as well.

There's also a security issue reported, but it seems that that is not actually an issue on our side at all.

> As can be seen here, getting a foot back into fedora should be a major goal.

That seems a good goal. Do you have any ties with Fedora or suggestions on what we could do to facilitate that?

**Comment by petrovo-as** [comment]
Hi,

couldn't release a minor fix. Archlinux still only takes 3.0.3.tar.gz and there is a missing commit: [f14e427](https://github.com/projecthamster/hamster/commit/f14e4276d11c5d69fb6da844735cffc7a718f1f1)

Thanks
Petr
